src/fu-util.c

```diff
@@ -1968,6 +2011,8 @@ fu_util_modify_config (FuUtilPrivate *priv, gchar **values, GError **error)
 	if (!fu_systemd_unit_stop (fu_util_get_systemd_unit (), error))
 		return FALSE;
 #endif
+	/* TRANSLATORS: success message -- a per-system setting value */
+	g_print ("%s\n", _("Successfully modified configuration value"));
```

Comment:
For this one, we know what value was changed, should we reference it?

Reply:
I'm not convinced:

    $ sudo fwupdmgr modify-config --force IdleTimeout 7201
    Restart the daemon to make the change effective? [Y|n]: n

The thing we're changing and the value is literally two lines above.

Reply:
Oh yeah, I see, good point.